packages/xdl/src/start/LoadingPageHandler.ts

```diff
@@ -21,14 +23,22 @@ export function setOnDeepLink(listener: OnDeepLinkListener) {
   onDeepLink = listener;
 }
 
-function getPlatform(query: { [x: string]: string | string[] | null }): string | null {
+function getPlatform(query: { [x: string]: string | string[] | null }): 'android' | 'ios' | null {
   if (query['platform'] === 'android' || query['platform'] === 'ios') {
     return query['platform'];
   }
 
   return null;
 }
 
+function getRuntimeVersion(exp: ExpoConfig, platform: 'android' | 'ios' | null) {
+  if (!platform) {
+    return 'Undetected';
+  }
+
+  return getRuntimeVersionNullable(exp, platform) ?? 'Undetected';
```

Comment:
could we show the SDK version (and have the page say "SDK version") instead of "Undetected" if the runtime version is null?

Reply:
Will do it in the separate PR ;)